edge/cmd/edgecore/app/server.go

```diff
@@ -90,15 +91,30 @@ offering HTTP client capabilities to components of cloud to reach HTTP servers r
 			// Get edge node local ip only when the custiomInterfaceName has been set.
 			// Defaults to the local IP from the default interface by the default config
 			if config.Modules.Edged.CustomInterfaceName != "" {
-				klog.Infof("Getting IP address by custom interface: %s", config.Modules.Edged.CustomInterfaceName)
 				ip, err := netutil.ChooseBindAddressForInterface(config.Modules.Edged.CustomInterfaceName)
 				if err != nil {
-					klog.Errorf("Failed to get IP address by custom interface: %s: %s", config.Modules.Edged.CustomInterfaceName, err)
+					klog.Errorf("Failed to get IP address by custom interface %s: %s", config.Modules.Edged.CustomInterfaceName, err)
 					os.Exit(1)
 				}
 				config.Modules.Edged.NodeIP = ip.String()
+				klog.Infof("Get IP address by custom interface %s: %s", config.Modules.Edged.CustomInterfaceName, config.Modules.Edged.NodeIP)
+			} else {
+				ip := net.ParseIP(config.Modules.Edged.NodeIP)
+				if ip != nil {
+					klog.Infof("Get IP address from config: %s", config.Modules.Edged.NodeIP)
```

Comment:
```suggestion
                                        config.Modules.Edged.NodeIP = ip
					klog.Infof("Get node IP address from config successfully: %s", config.Modules.Edged.NodeIP)
```

Reply:
The ip should be used.

Reply:
> The ip should be used.
thanks for suggestions，the ip is parsed from config.Modules.Edged.NodeIP, so there is no need to assign again here

Reply:
Sounds good.

Reply:
```suggestion
					klog.Infof("Use node IP address from config: %s", config.Modules.Edged.NodeIP)
```